etc/schema.json

```diff
@@ -6792,6 +6840,9 @@
                 },
                 "ja4": {
                     "type": "string"
+                },
+                "ja4s": {
+                    "type": "string"
```

Comment:
Is there not some license question around ja4s ?

Reply:
Adding the label [decision-required] about this point

Reply:
last time I checked we can't directly add ja4s for license reasons indeed

Reply:
Its not just license, but patents as well. Even with the pass on the non-plus versions, we still want to gate it as a compile time option.